release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -732,7 +732,9 @@ For more information, see xref:../scalability_and_performance/cnf-performance-ad
 [id="ocp-4-7-dev-exp"]
 === Developer experience
 
-
+==== Red Hat OpenShift GitOps (Technology Preview)
+The Red Hat OpenShift GitOps 1.0 Technology Preview release introduces a declarative way to implement continuous deployment for cloud native applications. You can use Red Hat OpenShift GitOps to adopt GitOps principles for managing cluster configurations, and automating secure, predictable, traceable, and repeatable application delivery across hybrid, multi-cluster, Kubernetes environments.
```

Comment:
Something about the intro to the second clause is confusing - do you mean:
`You can use Red Hat OpenShift GitOps to adopt GitOps principles for managing cluster configurations, and to automate secure, predictable, traceable, and repeatable application delivery across hybrid, multi-cluster, Kubernetes environments.`
or
`You can use Red Hat OpenShift GitOps to adopt GitOps principles for managing cluster configurations, and for automating secure, predictable, traceable, and repeatable application delivery across hybrid, multi-cluster, Kubernetes environments.`

Reply:
I would say the second, let me make the change and squash the commits